### PR TITLE
feat(node/_fs): Add `fs.writev(fd, buffers[, position], callback)`

### PR DESCRIPTION
**Pull request by getspooky**
Ref https://github.com/denoland/deno_std/issues/1863

### CODE REVIEW

File: node/_fs/_fs_writev.mjs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      currentOffset += await Deno.writeSync(fd, buffer.subarray(currentOffset));
+    }
+    return currentOffset - offset;
+  };
+
+  fd = getValidatedFd(fd);
+  validateBufferArray(buffers);
+  callback = maybeCallback(callback || position);
+
+  if (buffers.lenght === 0) {
```
Comment (kt3k) [inline node/_fs/_fs_writev.mjs:34]: `lenght` -> `length`

**Comment by kt3k** [comment]
@getspooky Thank you for working on this! Can you add some test cases?

**Comment by kt3k** [comment]
Please merge main branch when #2012 is landed. That should fix the CI checks

**Comment by getspooky** [comment]
@kt3k can pls merge that, i need it to continue working on `fs.writevSync(fd, buffers[, position])`

**Review by kt3k — APPROVED** [review]
LGTM. Nice work!

**Pull request merged**